dev_utils/compose-no-tls.yml

```
@@ -19,18 +19,24 @@ services:
     volumes:
       - /tmp/data:/data
   mq:
-    image: neicnordic/sda-mq:v1.4.0
+    image: neicnordic/sda-mq:v1.4.24
```

```suggestion
    image: ghcr.io/neicnordic/sda-mq:v1.4.24
```

just to keep it consistent